README.md

```diff
@@ -1,13 +1,299 @@
+made by the people in the Elixir, Phoenix and Elm communities!
+These technologies are set to "***take off***" in 2017 and we are _excited_
+to be _sharing_ the technical/competitve advantage with our clients!
+
+## What?
+
+This document + diagram _describes_ the full "**PETE**" Technology Stack
+we use for dwyl products and services (clients).
+
+Each element in our stack was _carefully_ selected based
+on it's individual merits. <br />
+When _assembled_ into a seamless "machine",
+the stack _unrivaled_ for developer productivity and word-class quality!
+
+### Tutorials!
+
+We have _crafted_ a "***Complete Beginner's Guide***"
+for each element in the stack, so that we:
++ ***Document our collective learning***
+`while` we are building projects.
+(_because as humans we **forget fast** unless we **write it**_!)
++ ***Share*** our knowledge with other people so we can
+  + Help to train (_potential_) new team members
+  as quickly/effectively as possible.
+  + ***Collectively iterate*** on our knowledge and "_level-up_" as a _team_!
+  + "Onboard" the client team (_who may want/need_) to
+  support/maintain the codebase/project if/when we _seamlessly_ "hand over".
+  + Inform the wider community of both technical _and_ non-technical
+  people ("stake holders") who are _generally_
+  interested in _understanding_ the project.
+  + Enlighten other teams/organisations/agencies/etc. we aren't in
+  _direct_ contact with that there is a "_more fun_" way of building software!
++ Make _everyone's_ life easier/better
+by having a "launch pad" for
+[_rapid_ learning](https://youtu.be/hOZnP4dZYK0 "Matrix Easter Egg ;-)")!
+
+> For an _example_ project where this "Stack" is _implemented_
+see: https://github.com/healthlocker/healthlocker
+
+### The "PETE" Stack
+
+![dwyl-pete-stack](https://cloud.githubusercontent.com/assets/194400/23000461/8688abcc-f3d8-11e6-86ab-804720619d8e.png)
+
+"PETE" is an acronym<sup>1</sup> for the following elements:
+
++ **Phoenix** is a Web Application Framework that does not compromise
+on speed, reliability or maintainability!
+Phoenix is the "_successor_"
+to the incredibly popular "Ruby-on-Rails" framework.
+Built _from scratch_ by highly experienced engineers
+who worked on/with Rails. It _solves_
+all of the speed/socket/scaling/concurrency, issues
+people felt when building/using Rails apps.
+The list of ***benefits*** Phoenix has over
+(_vertually every_) other Web Frameworks is _extensive_.<br />
+Please see:
+[github.com/dwyl/learn-phoenix-framework#our-**top-10-reasons**-why-phoenix](https://github.com/dwyl/learn-phoenix-framework#our-top-10-reasons-why-phoenix) <br />
+
++ **Elixir** is the _functional_ programming language used to
+build using the Phoenix framework. Elixir is a _beautiful_ language
+written _from scratch_ to be ***friendly, concise and efficient***.
+***Yes***, Elixir not a "_popular_" as JavaScript, Java, C# or PHP,
+but its' adoption is growing rapidly and most importantly its'
```

Comment:
I never know how to write `it's` or `its'` ... 
would be stoked for anyone with _much_ betterererer engrish to fix the typos in the next PR. 👍